src/libraries/Microsoft.Win32.SystemEvents/src/Microsoft.Win32.SystemEvents.csproj

```diff
@@ -104,6 +104,8 @@ Microsoft.Win32.SystemEvents</PackageDescription>
              Link="Common\Interop\Windows\WtsApi32\Interop.WTSUnRegisterSessionNotification.cs" />
     <Compile Include="$(CommonPath)Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs"
              Link="Common\Microsoft\Win32\SafeHandles\SafeLibraryHandle.cs" />
+    <Compile Include="$(CommonPath)System\Obsoletions.cs"
+             Link="Common\System\Obsoletions.cs" />
```

Comment:
This shouldn't be necessary when you set `<IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>` in this project.

Reply:
There are build errors if this is removed, even with `<IncludeInternalObsoleteAttribute>true</IncludeInternalObsoleteAttribute>`

Reply:
@ViktorHofer I'm following https://github.com/dotnet/runtime/blob/main/docs/project/list-of-diagnostics.md#obsoletions which seems to recommend including this line. Without it I will get build error like in the link above, so I've re-included it. The recommendation in my build error now will cause changes to be made in CompatibilitySuppressions.xml like I had [earlier](https://github.com/dotnet/runtime/pull/108489#discussion_r1815521176), is this expected next step or do you have recommendation on a way to get around this without changing CompatibilitySuppressions.xml?

Reply:
The documentation page seems to be misleading:

![image](https://github.com/user-attachments/assets/d14054f9-336d-42ba-b30d-7893a15538c9)

It mentions both actually but I think that the property should be enough. cc @jeffhandley

Reply:
OK I misread. The Obsoletions.cs contain the obsoletion messages and `<IncludeInternalObsoleteAttribute>` includes the internal API for downlevel support. The docs are right.